docs/proposals-accepted/20240909-live-blocks-in-objectstorage.md

```diff
@@ -0,0 +1,62 @@
+---
+type: proposal
+title: Avoid querying blocks in object storage if they have a live source
+status: approved
+owner: MichaHoffmann
+menu: proposals-accepted
+---
+
+## Avoid querying blocks in object storage if they have a live source
+
+> TL;DR: We propose to not query blocks in object storage that are still served by a Sidecar/Ruler or Receiver.
+
+## Why
+
+Accessing blocks from object storage is generally orders of magnitude slower then accessing the same data from other sources that can query them from disk or from memory. If the source that originally uploaded the blocks is still alive and still has access to the block then we would throw away the expensively obtained data from object storage during deduplication.
```

Comment:
I would provide a config example here, where this holds true where a source still has access to block and it also being available in storegw

Reply:
Added a small example for the situation i try to address